### ISSUE TITLE
Expand/collapse icons in some Blazor components do not have cursor: pointer

### ISSUE THREAD

**Issue by ntacheva** [telerik/kendo-themes#3433]
**Describe the bug**

In UI for Blazor hovering over the expand/collapse icons of some components displays the default cursor instead of pointer. For example, TreeList, Gantt, grouped Grid.

The issue stems from the lack of a `k-widget` class in Blazor components that targets the cursor pointer in the themes:

https://github.com/telerik/kendo-themes/blob/master/packages/default/scss/icons/_layout.scss#L103

**To reproduce**

1. Go to the [TreeList demo](https://demos.telerik.com/blazor-ui/treelist/overview), [Gantt demo](https://demos.telerik.com/blazor-ui/gantt/overview), [Grid Grouping demo](https://demos.telerik.com/blazor-ui/grid/grouping) (group the Grid by some field to see the icons) or [Grid Hierarchy demo](https://demos.telerik.com/blazor-ui/grid/hierarchy)
1. Hover some expand/collapse icon
1. The default cursor is shown while it should be a pointer one as in the [TreeView component](https://demos.telerik.com/blazor-ui/treeview/overview)

**Expected behavior**

Hovering over the expand/collapse icons in the components should display cursor: pointer instead of the default one.

**Affected package (please remove the unneeded items)**

* All

**Affected suites (please remove the unneeded items)**

* Telerik UI for Blazor

**Affected browsers (please remove the unneeded items)**
* All

**Comment by jivanova** [comment]
@TeyaVes I thought that we are moving away from the "k-widget" class. If that is correct, I believe we should define the rendering in this item and sync all teams. Either `k-cursor-pointer` or another class is a good approach for us. We should add it in packages html, or in a test once decided so that everyone will have it as a reference.

**Comment by TeyaVes** [comment]
@jivanova , yes, you are correct, we are moving away from the 'k-widget' class. Once I add the needed wrapper in the packages/html, I'll update the issue here as well.
